### DISCUSSION TITLE
Manual for Tandoor App available?

### DISCUSSION THREAD

**Discussion by FreedomIsAChoice** [TandoorRecipes/recipes#2943] [Q&A]
Hello,
is there anywhere a manual available where I can search for any questions using Tandoor?
For example, I want to create some additional users, but I can't find this function.
But I think this is not the only question which I need to know... 😉

Best regards

**Comment by smilerz** [comment]
the most complete documentation is [here](https://docs.tandoor.dev/).

You can invite users at Space Settings

**Comment by FreedomIsAChoice** [comment]
Thank you @smilerz for the link to the documentation.
There is no section for a user management.
Im my space settings I found the possibility to invite users, but a token is needed.
Where do I generate a token in the admin settings?

**Reply by smilerz** [reply]
I think the process of creating an invite generates the token.
